blobfuse2-release.yaml

```diff
@@ -527,3 +532,43 @@ stages:
                   changeLogType: 'commitBased'
                   isDraft: false
                   assetUploadMode: replace
+  - stage: UpdateLatestVersion
+    dependsOn: ReleaseArtifact
+    condition: succeeded('ReleaseArtifact')
+    jobs:
+      - job: UpdateVersion
+        pool:
+          vmImage: 'ubuntu-18.04'
+        variables:
+          - group: NightlyBlobFuse
+        
+        steps:
+          - checkout: none
+
+          - script: |
+              echo ${{ parameters.tag }}
+            displayName: 'Tag Name'
+
+          - ${{ if eq(parameters.update_version, true) }}:
+              - script: |
+                  sudo apt install python3
+                  python3 --version
+                displayName: 'Installing Python'
+
+              - script: |
+                  python3 -c "import json; import requests; import sys;
+                  if(len(sys.argv)!=3):
+                      print('Less arguments')
+                      sys.exit(1)
+                  url = sys.argv[1]
+                  version = sys.argv[2].replace('blobfuse2-', '')
+                  if(len(version)==0):
+                      print('Incorrect Release Tag')
+                      sys.exit(1)
+                  dataDict = requests.get(url).json()
+                  dataDict['blobfuse2'] = version
+                  resp = requests.put(url, data=json.dumps(dataDict, indent=4), headers={'Content-Type': 'application/json', 'x-ms-blob-type': 'BlockBlob'})
+                  print('Status Code: ' + str(resp.status_code))
+                  sys.exit(1) if(resp.status_code<200 or resp.status_code>202) else print('SUCCESS')" \
+                  '$(LATEST_VERSION_SAS)' '${{ parameters.tag }}'
```

Comment:
We are relying on the tag here that it will always be in sync with the binary version.

Reply:
Done

Reply:
We could also somehow read it from the code

Reply:
Removed the dependency from the tag that we specify during the build.
Now it will install blobfuse2 and pass the version as a parameter to the python script from the **blobfuse2 version** command